dedupe/ckan_api.py

```diff
@@ -111,26 +111,26 @@ def get_dataset(self, organization_name, identifier, is_collection, sort_order='
 
         return results[0]
 
-    def get_duplicate_identifiers(self, organization_name, is_collection):
+    def get_duplicate_guids(self, organization_name, is_collection):
         filter_query = 'organization:"%s" AND type:dataset' % organization_name
         if is_collection:
             filter_query = '%s AND collection_package_id:*' % filter_query
 
         response = self.get('/3/action/package_search', params={
             'fq': filter_query,
-            'facet.field': '["identifier"]',
+            'facet.field': '["guid"]',
             'facet.limit': -1,
             'facet.mincount': 2,
             'rows': 0,
             })
 
         return \
-            response.json()['result']['search_facets']['identifier']['items']
+            response.json()['result']['search_facets']['guid']['items']
 
-    def get_dataset_count(self, organization_name, identifier, is_collection):
+    def get_dataset_count(self, organization_name, guid, is_collection):
         filter_query = \
-            'identifier:"%s" AND organization:"%s" AND type:dataset' % \
-            (identifier, organization_name)
+            'guid:"%s" AND organization:"%s" AND type:dataset' % \
+            (guid, organization_name)
```

Comment:
Leave the identifier here as identifier. Add a new optional parameter called identifier_type that defaults to identifier, and then use that parameter in place of the first identifier on line 132.